lazy-reducer-enhancer.js

```diff
@@ -28,20 +28,18 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
     someReducer
   });
 */
+
 export const lazyReducerEnhancer = function(combineReducers) {
   return (nextCreator) => {
     return (origReducer, preloadedState) => {
       let lazyReducers = {};
```

Comment:
@notwaldorf Sorry to ask this on a closed PR but what is the purpose of this variable?